tests/testsuite/cargo_information/within_ws_and_pick_ws_package/stdout.log

```diff
@@ -0,0 +1,7 @@
+cargo-list-test-fixture
+version: 0.2.0
+license: unknown
+rust-version: unknown
+dependencies:
+  my-package@0.1
+note: to see how you depend on cargo-list-test-fixture, run `cargo tree --package cargo-list-test-fixture@0.2.0 --invert`
```

Comment:
I don't think we should use the cargo tree command to check the main package itself as it might be a bug(or useless tip).

Reply:
Within a large workspace, it could still be relevant information

Reply:
Could you please provide an example? I have tried it in several projects and different package directories, but it seems that they do not have this type of dependency relationship. It only prints the package itself.

Reply:
... thats annoying.

To get it to work, you have to pick both the begin and end point, like `cargo tree -i clap -p clap_complete`